src/index.html

```diff
@@ -0,0 +1,240 @@
+       <div id="accordion">
+          <div class="card">
+             <div class="card-header">
+                <a class="card-link" data-toggle="collapse" href="#collapseOne">
+                HTML-Prototyping mit Baumeister Teil 1 &#x2013; Installation
+                </a>
+             </div>
+             <div id="collapseOne" class="collapse show" data-parent="#accordion">
+                <div class="card-body">
+                   <div style="padding:56.25% 0 0 0;position:relative"><iframe src="https://player.vimeo.com/video/268572516?color=75B7DD&amp;title=0&amp;byline=0&amp;portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen></iframe></div>
+                   <script src="https://player.vimeo.com/api/player.js"></script>
+                </div>
+             </div>
+          </div>
+          <div class="card">
+             <div class="card-header">
+                <a class="collapsed card-link" data-toggle="collapse" href="#collapseTwo">
+                HTML-Prototyping mit Baumeister Teil 2 &#x2013; Erste Schritte nach der Installation
+                </a>
+             </div>
+             <div id="collapseTwo" class="collapse" data-parent="#accordion">
+                <div class="card-body">
+                   <div style="padding:56.25% 0 0 0;position:relative"><iframe src="https://player.vimeo.com/video/268574813?color=75B7DD&amp;title=0&amp;byline=0&amp;portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen></iframe></div>
+                   <script src="https://player.vimeo.com/api/player.js"></script>
+                </div>
+             </div>
+          </div>
+          <div class="card">
+             <div class="card-header">
+                <a class="collapsed card-link" data-toggle="collapse" href="#collapseThree">
+                HTML-Prototyping mit Baumeister Teil 3 &#x2013; Handlebars Templates
+                </a>
+             </div>
+             <div id="collapseThree" class="collapse" data-parent="#accordion">
+                <div class="card-body">
+                   <div style="padding:56.25% 0 0 0;position:relative"><iframe src="https://player.vimeo.com/video/268576053?color=75B7DD&amp;title=0&amp;byline=0&amp;portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen></iframe></div>
+                   <script src="https://player.vimeo.com/api/player.js"></script>
+                </div>
+             </div>
+          </div>
+       </div>
+    </div>
+ </div>
+ <div class="row">
+    <div id="anchor-install" class="install gray col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>Quick install guide</h2>
+       <h6>For those already using Node, Gulp and stuff.</h6>
```

Comment:
Baumeister verwendet kein Gulp mehr. Von daher besser:

```suggestion
       <h6>For those already using Node and npm.</h6>
```

Reply:
Done